lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -460,6 +460,11 @@ func convertContainerToContainerInfo(container *exec.ContainerInfo) *models.Cont
 		info.ProcessConfig.StartTime = session.StartTime
 		info.ProcessConfig.StopTime = session.StopTime
 
+		if session.Group != "" {
+			info.ProcessConfig.User = fmt.Sprintf("%s:%s", session.User, session.Group)
+		} else {
+			info.ProcessConfig.User = session.User
```

Comment:
[suggestion] this could be
```
info.ProcessConfig.User = session.User
if session.Group != "" {
    info.ProcessConfig.User = fmt.Sprintf("%s:%s", session.User, session.Group)
 } 
			
```